src/components/bulkConcepts/addBulkConcepts.jsx

```diff
@@ -85,6 +85,7 @@ export class AddBulkConcepts extends Component {
     const lastConcept = indexOfFirstConcept + currentConcepts.length;
     const firstConcept = indexOfLastConcept - 19;
     const { type, typeName, collectionName, language } = this.props.match.params;
+    const disableButton = ((this.state.cielConcepts.length === 0) || isFetching);
```

Comment:
@ozezzy, I would be best if you destructured state here too.

Reply:
@abulojoshua1 the component uses two "cielConcepts" one from props and the other in component state. one has been distructured from the props. I can't distructure the second from state in the same scope thats why I referenced it directly. Let me know if there is another way